src/openrct2/util/SawyerCoding.cpp

```diff
@@ -42,7 +43,7 @@ uint32_t sawyercoding_calculate_checksum(const uint8_t* buffer, size_t length)
  */
 size_t sawyercoding_write_chunk_buffer(uint8_t* dst_file, const uint8_t* buffer, sawyercoding_chunk_header chunkHeader)
 {
-    uint8_t *encode_buffer, *encode_buffer2;
+    std::unique_ptr<uint8_t> encode_buffer, encode_buffer2;
```

Comment:
These are pointing to arrays, therefore they must be declared as `std::unique_ptr<uint8_t[]>`